src/ui/src/js/controllers/system_view.js

```diff
@@ -18,13 +18,21 @@ export default function systemViewController(
     UtilityService,
     DTOptionsBuilder,
     system) {
-      
-  $scope.util = UtilityService;
 
+
+
+  $scope.util = UtilityService;
+  $scope.filterHidden = false;
+  $scope.tableS;
```

Comment:
Is `$scope.tableS;` necessary?

Reply:
It is not necessary was using to try to map out rerender or redraw functions.